benchmarks/benchmarks/stats.py

```diff
@@ -92,5 +92,15 @@ def time_distribution(self, distribution, properties):
                 stats.beta.rvs(size=1000, a=5, b=3, loc=4, scale=10)
             elif properties == 'fit':
                 stats.beta.fit(self.x, a=5, b=3, loc=4, scale=10)
-        
+
+
+class DescriptiveStats(Benchmark):
+    def setup(self):
```

Comment:
Probably clearer to [parametrize](http://asv.readthedocs.io/en/latest/writing_benchmarks.html#parameterized-benchmarks) here -- you can make setup accept an argument for the size of the data set to be used and then the tests will be categorized accordingly with their performance. i.e., setup(self, n_points).